config/main.py

```diff
@@ -1770,21 +1771,28 @@ def startup(ctx, interface_name):
         if interface_name is None:
             ctx.fail("'interface_name' is None!")
 
-    if interface_name_is_valid(interface_name) is False:
-        ctx.fail("Interface name is invalid. Please enter a valid interface name!!")
+    intf_fs = parse_interface_in_filter(interface_name)
+    if len(intf_fs) == 1 and interface_name_is_valid(interface_name) is False:
```

Comment:
Shouldn't we check for validity of each interface in the list?

Reply:
in case of len(intf_fs) == 1 meaning user only gives one single interface_name
If there are multiple interfaces in intf_fs, only existing interfaces will be taken later.

Reply:
Can you explain on where is it taken later?

Reply:
I mean if user give a single interface name say "Ethernet0" then it will be checked at line 1775. Otherwise, the interface names will be converted to a list intf_fs and will be checked at line 1781, 1786 etc.

Reply:
@stevenlu99 , i dint mean the check in 1781. Was referring to `interface_name_is_valid`. Is that handled if you've multiple interfaces with comma seperated?

Reply:
interface_name_is_valid will only check for single interface name. and that's why I had "len(intf_fs) == 1"